capture_gui/plugins/displayplugin.py

```diff
@@ -128,7 +128,11 @@ def display_gradient(self):
         define the gradient otherwise the background color will be used as 
         solid color.
         """
-        return self.display_type.currentText() == "Gradient"
+        state = True
```

Comment:
I am not 100% sure but I think it is a left over of figuring out why the gradient setting got overwritten.
Less code is less errors, I will revert.